steelthread-0/walkthrough.md

```diff
@@ -25,62 +28,68 @@ You should also take the opportunity to login your oc CLI as this user. You can
 
 # Setting up a Enmasse queue in your namespace 
 
+- From your namespace click the catalog button on the left hand nav
 - Provision a Enmasse service from the catalog (choose EnMasse standard) name it eval and choose not to bind at this time.
 - Once complete, in the Eval namespace you will find a EnMasse provisioned service.
-- Create a binding to this provisioned service checking each available option.
+- Now create a binding to this provisioned service checking each available option.
```

Comment:
@maleck13 These steps can be simplified/removed by doing this card https://trello.com/c/8cBD7QvP/106-update-the-steel-thread-0-integration-steps-to-use-an-addressspace-custom-resource-instead-of-using-the-service-catalog